docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md

```diff
@@ -45,7 +45,7 @@ Table 2 lists the limits and describes the differences in the portion of the ite
 
 **Table 2. Limits on the size of the item body evaluated**
 
-||Outlook rich client|Outlook on mobile devices|Outlook on the web|
+|&mdash;|Outlook rich client|Outlook on mobile devices|Outlook on the web|
```

Comment:
@ElizabethSamuel-MSFT I found a few tables in this style, where the first (top left) cell was empty. Curious what you think of the approach I chose.

Reply:
Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36413), we shouldn't use an em dash either. However, tables where the first column has no title but represents disparate info are considered data tables and this is expected (see [contributor guide](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?toc=%2Fhelp%2Fplatform%2Ftoc.json&bc=%2Fhelp%2Fplatform%2Fbreadcrumb%2Ftoc.json&branch=main#data-matrix-tables)). As such, it should be ok to leave it blank. That said, I don't know what the narrator experience is.